### PR TITLE
Build img consolidation and streamline

### PR DESCRIPTION
**Pull request by mazzystr**
Build img consolidation and streamline.  We now build a single img containing mkdocs, yaspeller, htmlproofer packed in.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -91,10 +91,59 @@ ifdef SELINUX_ENABLED
...
+	@echo '*** END cat mkdocs.yml ***'
+	mkdocs build -f mkdocs.yml -d site
+
+
+## Build image localhost/kubevirt-user-guide
+build_img: | envvar
+	@echo "${GREEN}Makefile: Building Image ${RESET}"
+	${DEBUG}if [ ! -e "./Dockerfile" ]; then \
+	  IMAGE="`echo $${IMGTAG} | sed -e s#\'##g -e s#localhost\/## -e s#:latest##`";  \
+	  if [ "`curl https://raw.githubusercontent.com/kubevirt/project-infra/master/images/kubevirt-user-guide/Dockerfile -o ./Dockerfile -w '%{http_code}\n' -s`" != "200" ]; then \
```
Comment (dhiller) [inline Makefile:131]: Why not use `--fail` here?

**Review by dhiller — APPROVED** [review]
General lgtm, one nit inline. Decide for yourself whether you just want to leave it like it is.
/approve
/hold

File: Makefile (hunk 1 above)
Comment (mazzystr) [reply Makefile]: Excellent idea!  I also modified the other curl as well as the curls in website Makefile

**Comment by mazzystr** [comment]
@dhiller I'm ready for merge.  [project-infra/pull/1230](https://github.com/kubevirt/project-infra/pull/1230) will need to be approved/merged along with this one.

**Review by dhiller — APPROVED** [review]
/unhold

Thanks!

**Pull request merged**